packages/blockchain/test/customConsensus.spec.ts

```diff
@@ -150,21 +153,5 @@ describe('consensus transition checks', () => {
         `checkAndTransitionHardForkByNumber should not throw with custom consensus, error=${err.message}`
       )
     }
-
-    blockchain.consensus = new EthashConsensus()
-    blockchain.common.consensusAlgorithm = () => 'fibonacci'
-
-    try {
-      await blockchain.checkAndTransitionHardForkByNumber(5n)
```

Comment:
This part of the test got deleted, is this correct?

Reply:
Could readd by making the checks more consistent.